apps/main/src/bridge/components/PageBridges/components/BridgeOverview.tsx

```diff
@@ -0,0 +1,59 @@
+import Fuse from 'fuse.js'
+import { useMemo, useState } from 'react'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import { useSwitch } from '@ui-kit/hooks/useSwitch'
+import { t } from '@ui-kit/lib/i18n'
+import { TableSearchField } from '@ui-kit/shared/ui/DataTable/TableSearchField'
```

Comment:
Perhaps it's a bit weird to import it from the datatable, but it checks all boxes and isn't actually coupled to tanstack-table itself. Maybe we need to move it around in ui-kit but didn't want to bother just yet.